### PR TITLE
Fix context menu for Image Edit menu

### PR DESCRIPTION
**Pull request by juliaroldi**
To make context menu work with image edit plugin, when the user right clicks on an image, the editing operations will be applied, and the image will be selected. Also, if the image was selected and right click occurs in an element that is not a image, remove image selection. 
ContextMenuOnImages

### CODE REVIEW

**Comment by BryanValverdeU** [comment]
Hey Julia, I think this looks good. is Shift + F10 to open the context menu should also be supported?

File: packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts [hunk 1]
```diff
@@ -143,14 +144,31 @@ export class ImageEditPlugin implements ImageEditor, EditorPlugin {
...
+            }
+            if (
+                isElementOfType(target, 'span') &&
+                target.firstElementChild &&
+                isNodeOfType(target.firstElementChild, 'ELEMENT_NODE') &&
+                isElementOfType(target.firstElementChild, 'img')
+            ) {
+                return true;
+            }
+            return false;
```
Comment (BryanValverdeU) [inline packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts:160]: nit: I think you can replace this with: 

```
           return (isElementOfType(target, 'img')) || (
                isElementOfType(target, 'span') &&
                target.firstElementChild &&
                isNodeOfType(target.firstElementChild, 'ELEMENT_NODE') &&
                isElementOfType(target.firstElementChild, 'img')
            )
```

File: packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts [hunk 2]
```diff
@@ -97,11 +98,11 @@ export class ImageEditPlugin implements ImageEditor, EditorPlugin {
...
-                        this.applyFormatWithContentModel(
-                            this.editor,
-                            this.isCropMode,
-                            true /* shouldSelectImage */
-                        );
+                        // this.applyFormatWithContentModel(
+                        //     this.editor,
+                        //     this.isCropMode,
+                        //     true /* shouldSelectImage */
+                        // );
```
Comment (BryanValverdeU) [inline packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts:105]: Should we remove this?

Comment (juliaroldi) [reply packages/roosterjs-content-model-plugins/lib/imageEdit/ImageEditPlugin.ts:105]: I was using to debug and forgot to uncomment it!

**Comment by juliaroldi** [comment]
> Hey Julia, I think this looks good. is Shift + F10 to open the context menu should also be supported?

I didn't test it, at least for mac, this shortcut controls volume

**Comment by BryanValverdeU** [comment]
> > Hey Julia, I think this looks good. is Shift + F10 to open the context menu should also be supported?
> 
> I didn't test it, at least for mac, this shortcut controls volume

I see for Window it also triggers the context menu.

**Comment by juliaroldi** [comment]
> > > Hey Julia, I think this looks good. is Shift + F10 to open the context menu should also be supported?
> > 
> > 
> > I didn't test it, at least for mac, this shortcut controls volume
> 
> I see for Window it also triggers the context menu.

It will not trigger, because only modifier key selects the image, let me add shift to select the image too

**Review by BryanValverdeU — APPROVED** [review]

**Comment by BryanValverdeU** [comment]
Shift + F10 is working now:
Fixed

**Review by BryanValverdeU — APPROVED** [review]

**Pull request merged**